WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -265,8 +265,12 @@ class GutenbergViewController: UIViewController, PostEditor {
         return GutenbergImageLoader(post: post)
     }()
 
+    private lazy var gutenbergSentry: GutenbergSentry = {
+        return GutenbergSentry()
+    }()
```

Comment:
This condensed syntax should work, too

```suggestion
    private lazy var gutenbergSentry = GutenbergSentry()
```

The result is the same, but I though it worth sharing because it's fun to see how the compiler unpacks the condensed syntax I suggested into the full one you used 😄